src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -31302,7 +31302,7 @@ static void Main(string[] args)
     {
         switch (true)
         {
-            case 1+TakeOutParam(3, out var x1):
+            case !TakeOutParam(3, out var x1):
```

Comment:
>! [](start = 17, length = 1)

Is this change necessary? (Does the test remain a valid regression test for #14717?)

Reply:
One or the other change is necessary to prevent it from looking like a positional pattern. I changed it different ways in two different branches, and resolved it to this one.

---
In reply to: [176458497](https://github.com/dotnet/roslyn/pull/25544#discussion_r176458497) [](ancestors = 176458497)